agent/stats/utils.go

```diff
@@ -35,9 +38,8 @@ func nan32() float32 {
 // dockerStatsToContainerStats returns a new object of the ContainerStats object from docker stats.
 func dockerStatsToContainerStats(dockerStats *docker.Stats) (*ContainerStats, error) {
 	// The length of PercpuUsage represents the number of cores in an instance.
-	numCores := uint64(len(dockerStats.CPUStats.CPUUsage.PercpuUsage))
-	if numCores == 0 {
-		seelog.Debug("Invalid container statitistics reported, got number of cores = 0")
+	if len(dockerStats.CPUStats.CPUUsage.PercpuUsage) == 0 {
```

Comment:
Do we still need this check here? When `dockerStats.CPUStats.CPUUsage.PercpuUsage` is `0`, do rest of the fields in the struct have invalid values as well? Would be good to run a `stress` workload and inspect the output when this happens.